repo/packages/M/mongodb-admin/0/resource.json

```diff
@@ -0,0 +1,19 @@
+{
+  "images": {
+    "icon-small": "http://www.file-extensions.org/imgs/app-icon/48/8854/mongodb-icon.png",
+    "icon-medium": "http://www.sudops.com/wp-content/uploads/2014/02/mongodb-96x96.png",
+    "icon-large": "http://1.bp.blogspot.com/-FagQLEI_Zhc/VnAzE9L8e3I/AAAAAAAAAFk/DWPqvL0ZXo8/s320/mongodb.jpg",
```

Comment:
Image assets should be hosted on downloads.mesosphere.com because these locations are not reliable and can change anytime. I haven't done this myself but I'm sure @pyronicide can tell us how :)